modular_skyrat/modules/exp_corps/code/expeditionary_trooper.dm

```diff
@@ -0,0 +1,121 @@
+/datum/job/expeditionary_trooper
+	title = "Expeditionary Trooper"
+	department_head = list("Captain")
+	faction = "Station"
+	total_positions = 2
+	spawn_positions = 2
+	supervisors = "the captain"
+	selection_color = "#ffeeff"
+	minimal_player_age = 40
+	exp_requirements = 400
+	exp_type = EXP_TYPE_SCIENCE
+
+	outfit = /datum/outfit/job/expeditionary_trooper
+
+	paycheck = PAYCHECK_HARD
+	paycheck_department = ACCOUNT_SCI
+
+	display_order = JOB_DISPLAY_ORDER_EXP_CORPS
+	bounty_types = CIV_JOB_SCI
+
+	family_heirlooms = list(/obj/item/binoculars)
+
+	trusted_only = TRUE
+
+/datum/job/expeditionary_trooper/after_spawn(mob/living/carbon/human/H, mob/M)
```

Comment:
https://github.com/tgstation/tgstation/pull/56872

Reply:
can't change how VSC sets it up, H/M is fine.